docs/design/generics/details.md

```diff
@@ -3785,24 +3806,58 @@ external impl forall [T:! Printable] Vector(T) as Printable {
 }
 ```
 
-To include these `impl` definitions inline in a `class` definition, include a
-`forall` clause with a more-specific type between the `impl` and `as` keywords.
+Note that no `forall` clause or type may be specified when declaring an `impl`
+with the `extend` keyword:
 
 ```
 class Array(T:! type, template N:! i64) {
-  impl forall [P:! Printable] Array(P, N) as Printable { ... }
+  // ❌ Illegal: nothing allowed before `as` after `extend impl`:
+  extend impl forall [P:! Printable] Array(P, N) as Printable { ... }
 }
 ```
 
-All internal `impl` declarations in the body of a `class` definition must be for
-the class being defined. It is an error to declare `impl i32 as Printable`
-inside `class Array`.
+**Note:** This was changed in
+[proposal #2760](https://github.com/carbon-language/carbon-lang/pull/2760).
+
```

Comment:
```suggestion
```

I assume this was temporary while #2760 was in flight, since references are generally at the bottom.

Reply:
I think this comment is useful here in context, rather than somewhere else.